.gitignore

```diff
@@ -2,4 +2,5 @@
 
 # OSX Hidden Files
 .DS_Store
-._*
\ No newline at end of file
+._*
+jp2_pc/Build/*
```

Comment:
Please separate this into its own section under the comment `# OpenTrespasser`, like so:
```
# OSX Hidden Files
.DS_Store
._*

# OpenTrespasser
jp2_pc/Build/*
```

Reply:
Done